recipes/ncbi_refseq_accession_offsets/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ncbi_refseq_accession_offsets" %}
+{% set version = "2021.11.16" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/64/84/6e71ac3bd8aa091d6fbea545cda44e14bac0347841ad4085983bf891c9e5/{{ name }}-{{ version }}-py3-none-any.whl"
+  sha256: 6a21176232543998c967b88844ce5b77d6c4ca7a05c665558f678651e4454067
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install {{ name }}-{{ version }}-py3-none-any.whl --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - urllib3 >=1.25.8
+    - zstandard >=0.15.1
+  run:
+    - python
+    - urllib3 >=1.25.8
+    - zstandard >=0.15.1
+
+test:
+  imports:
+    - ncbi_refseq_accession_offsets
+
+about:
+  home: "https://pypi.org/project/ncbi-refseq-accession-lengths/"
```

Comment:
Is this project not hosted on GitHub or any other code hosting service?

Reply:
This library is part of a bigger one: https://github.com/taxoniq/taxoniq/tree/main/db_packages/ncbi_refseq_accession_lengths